api/analyzedphenotypes.systemvars.api.inc

```diff
@@ -0,0 +1,206 @@
+<?php
+/**
+ * @file
+ *
+ * API: SYSTEM VARIABLES.
+ * Manage property and operation pertaining to system variables.
+ */
+
+
+/**
+ * DATA SOURCE
+ * Define system variable names.
+ *
+ * @return
+ *   An array of strings, constructed variable names grouped by variable type (set).
+ */
+function ap_define_variablenames() {
```

Comment:
Define makes me think you're setting the variables... Perhaps `ap_construct_variablenames()` would be better?